pkg/utils/strings.go

```diff
@@ -45,3 +45,24 @@ func ContainString(finalizers []string, f string) bool {
 	}
 	return false
 }
+
+// AddOrSwapString Add an element to the string array,
+// swap it with the last element if it is in the array,
+// or append it directly to the end if it does not exist.
+// And give whether the swap occurred.
```

Comment:
if  follow this case `give whether the swap occured` ,

- `true` means `occured`
- `false` means not occured